client/state/selectors/is-site-on-business-plan.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import { isBusinessPlan } from 'lib/plans';
+
+/**
+ * Returns true if site is on a business plan, false if the site is not
+ * or if the site or plan is unknown.
+ *
+ * @param { object } state Global state tree
+ * @param { number } siteId Site ID
+ * @returns { boolean } If the site is on a Business plan
+ */
+const isSiteOnBusinessPlan = ( state, siteId ) => {
+	const currentPlan = getCurrentPlan( state, siteId );
+
+	if ( ! currentPlan ) {
+		return false;
+	}
+
+	return isBusinessPlan( currentPlan.productSlug );
```

Comment:
We need to check for a Business plan or eCommerce plan

Reply:
The existing https://github.com/Automattic/wp-calypso/blob/57261aa3f0b3398e720a6ae91a6c03b88028e62c/client/state/selectors/is-site-on-atomic-plan.js should do the trick. So no need for a new selector.

Reply:
Ahhh thanks! :)